src/main/java/com/github/hcsp/classloader/MyClassLoader.java

```diff
@@ -25,7 +28,17 @@ public MyClassLoader(File bytecodeFileDirectory) {
     // 扩展阅读：ClassLoader类的Javadoc文档
     @Override
     protected Class<?> findClass(String name) throws ClassNotFoundException {
-        throw new ClassNotFoundException(name);
+        name = bytecodeFileDirectory.getPath() + "\\" + name + ".class";
+        System.out.println(name);
+        try (FileInputStream fileInputStream = new FileInputStream(new File(name)); ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream()) {
+            int i = 0;
+            while ((i = fileInputStream.read()) != -1) {
+                byteArrayOutputStream.write(i);
+            }
+            return defineClass(name, byteArrayOutputStream.toByteArray(), 0, byteArrayOutputStream.toByteArray().length);
+        } catch (IOException e) {
+            throw new ClassNotFoundException();
```

Comment:
```suggestion
            throw new ClassNotFoundException(e);
```


这个异常e不应该吞掉，而是应该作为`ClassNotFoundException`的原因继续抛出。